kuma/wiki/tests/test_views.py

```diff
@@ -193,9 +193,10 @@ def test_toc_view(self):
         ok_('Access-Control-Allow-Origin' in resp)
         eq_('*', resp['Access-Control-Allow-Origin'])
         self.assertHTMLEqual(
-            resp.content, '<ol><li><a href="#Head_2" rel="internal">Head 2</a>'
-            '<ol><li><a href="#Head_3" rel="internal">Head 3</a>'
-            '</ol></li></ol>')
+            resp.content, '<ol class="toc-links"><li>'
```

Comment:
This tests the ``$toc`` view, like https://developer.mozilla.org/en-US/docs/Web/CSS/display$toc. As far as I can tell, this is unused by Kuma and KumaScript, and is probably a good candidate for removal.

If you want to add the "toc-links" class to it, you'll need to change it here:

https://github.com/mozilla/kuma/blob/a8c812b60463c23e2c8ccfba73294d9f0bee1816/kuma/wiki/views/document.py#L407

Reply:
That page has broken HTML on it 😱 What fresh new can of worms have you found? 🐛